XamlControlsGallery/ControlPages/ProgressRingPage.xaml

```diff
@@ -37,7 +34,22 @@
             <local:ControlExample.Substitutions>
                 <local:ControlExampleSubstitution Key="IsActive" Value="{x:Bind ProgressToggle.IsOn, Mode=OneWay}" />
             </local:ControlExample.Substitutions>
-
+        </local:ControlExample>
+        <local:ControlExample HeaderText="A determinate progress ring.">
+            <StackPanel x:Name="Control2" Orientation="Horizontal">
+                <muxc:ProgressRing Width="130" x:Name="ProgressRing2" IsIndeterminate="False"/>
```

Comment:
Would also be good to understand what scenarios you expect apps to use a determinate ring for. The scenario should help influence the right size in the sample.

Reply:
Good idea! I'll update the page description so it's similar to progress bar.